### ISSUE TITLE
compile a version of UEDGE with OMFIT support on GA IRIS cluster

### ISSUE THREAD

**Issue by orso82** [LLNL/UEDGE#5]
@llnl-fesp @umansky I would like to compile a version of UEDGE with the OMFIT support (e4c491fdcb3a93d955e0cb3c6f02bae580fde8f7) on the GA IRIS cluster. I was given access to the `/fusion/projects/codes/uedge` folder on IRIS, but I wanted to coordinate with before making a new directory and starting to maintain a separate version of UEDGE just for usage via OMFIT.

Since @llnl-fesp was very prompt in merging e4c491fdcb3a93d955e0cb3c6f02bae580fde8f7 into master, I wonder if you could just update UEDGE on IRIS to the latest version :)

Please let me know how you'd like to proceed. Thanks!

**Comment by llnl-fesp** [comment]
I’m working on release now but it may be a couple of days. I’ll put what I have there now, the physics packages won’t change when I finally tag a release.

We still keep a basis version there but I know you mean pyUedge.

I’ll let you know the modulefile in a bit.

Bill

**Comment by llnl-fesp** [comment]
You can do a “ module load uedge/PyUEDGE/V7_08_04rc2”

Bill

**Comment by llnl-fesp** [comment]
Do you know the status of Python3 there at GA? On Iris there isn’t even a modulefile for python3.

Bill

**Comment by orso82** [comment]
Correct, there is no python3 installation that the computer group supports. However we support our own python3 installation for OMFIT. 
```
[meneghini@irisc ~]$ module load omfit/unstable
[meneghini@irisc ~]$ which python3
/fusion/projects/codes/atom/omfit_py3/atom_SATURN_GCC/miniconda3/bin/python3
```

I will get you the `forthon` and `mppl` packages installed as part of it...

**Comment by llnl-fesp** [comment]
No need. I already have mppl and forthon installed

**Comment by llnl-fesp** [comment]
If you want to install them into the python3 itself be my guest.  Forthon already was compatible with python3 and I just recently fixed the mppl at pypi.

**Comment by orso82** [comment]
@llnl-fesp both Forthon and mppl are now available with:
```
    module load omfit/unstable
    python3
```

**Comment by llnl-fesp** [comment]
Orso,

I stopped by to see you but heard you weren’t feeling well. Hope you are better for your travel tomorrow.

I just stopped by to have a short chat about PyUEDGE. I wanted to make sure that you knew that the variables you added to the hdf5 save file were fine for plotting but not for a restart. Using nis and ni as an example:

Nis is restored and copied to ni
Case is then run
If the case converged, then ni is copied to nis


Bill
